### PR TITLE
adding ProfileCard component

### PR DESCRIPTION
**Pull request by ZoltanT-RD**

### CODE REVIEW

File: FrontEnd/src/index.scss [hunk 1]
```diff
@@ -36,6 +34,26 @@ li {
...
+  margin-bottom: 0;
+}
+
+.ml0 {
+  margin-left: 0;
+}
+
+.mr0 {
+  margin-right: 0;
+}
```
Comment (ZoltanT-RD) [inline FrontEnd/src/index.scss:59]: these are not built-in, like in BS, so I've added a few

File: FrontEnd/src/index.scss [hunk 2]
```diff
@@ -1,23 +1,21 @@
-@import './colors.scss';
+@import "./colors.scss";
```
Comment (ZoltanT-RD) [inline FrontEnd/src/index.scss:1]: I've added a sass formatter (sibiraj-s.vscode-scss-formatter) extention to VSCode, and that thinks these should be double =$

File: FrontEnd/src/components/ProfileCard.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#This is the 'ProfileCard' component
+
+Some description here about what this component really does, and the scenarios below
+
+<Preview>
+  <Story name="basic state">
+      <ProfileCard
+        id={"test0"}
+        userName ={"StoryBook Tester"}
+        userProfilePicURL ={"https://placekitten.com/g/250/250"}
```
Comment (ZoltanT-RD) [inline FrontEnd/src/components/ProfileCard.stories.mdx:17]: got this from Rachel, back in the days =]

File: FrontEnd/src/index.scss (hunk 2 above)
Comment (SiantyTown) [reply FrontEnd/src/index.scss:1]: fair enough haha -- i wasn't too sure when i did it either!!

**Review by SiantyTown — APPROVED** [review]

**Pull request merged**